services/core/shared/core/compat/sdk_v5/schema/accounts.js

```diff
@@ -16,11 +16,11 @@
 module.exports = {
 	primaryKey: 'address',
 	schema: {
-		address: { type: 'string' },
-		publicKey: { type: 'string' },
+		address: { type: 'string', isDefaultColumn: true },
```

Comment:
Please consider renaming `isDefaultColumn` to `isDefaultSelect`, `isResultDefault` or `includeDefaultResult`.